pkg/deployments/tasks/20230206-aave-rebalanced-linear-pool-v4/input.ts

```diff
@@ -0,0 +1,31 @@
+import Task, { TaskMode } from '../../src/task';
+import { MONTH } from '@balancer-labs/v2-helpers/src/time';
+
+export type AaveLinearPoolDeployment = {
+  Vault: string;
+  BalancerQueries: string;
+  ProtocolFeePercentagesProvider: string;
+  WETH: string;
+  FactoryVersion: string;
+  PoolVersion: string;
+  InitialPauseWindowDuration: number;
+  BufferPeriodDuration: number;
+};
+
+const Vault = new Task('20210418-vault', TaskMode.READ_ONLY);
+const BalancerQueries = new Task('20220721-balancer-queries', TaskMode.READ_ONLY);
+const ProtocolFeePercentagesProvider = new Task('20220725-protocol-fee-percentages-provider', TaskMode.READ_ONLY);
+const WETH = new Task('00000000-tokens', TaskMode.READ_ONLY);
+
+const BaseVersion = { version: 4, deployment: '20230206-aave-rebalanced-linear-pool-v4' };
```

Comment:
This would be the name following the logic that we've been using in this repo, but let's double check.